_analyzers/token-graph.md

```diff
@@ -0,0 +1,201 @@
+
+    ```json
+    PUT /synonym_graph_index
+    {
+      "settings": {
+        "analysis": {
+          "filter": {
+            "my_synonyms": {
+              "type": "synonym_graph",
+              "synonyms": ["ssd => solid state drive"]
+            }
+          },
+          "analyzer": {
+            "my_analyzer": {
+              "tokenizer": "standard",
+              "filter": ["lowercase", "my_synonyms"]
+            }
+          }
+        }
+      },
+      "mappings": {
+        "properties": {
+          "content": {
+            "type": "text",
+            "analyzer": "my_analyzer"
+          }
+        }
+      }
+    }
+    ```
+    {% include copy-curl.html %}
+
+3. Create the same document in each index
+
+    ```json
+    PUT /synonym_index/_doc/1
+    { "content": "ssd is critical" }
+    ```
+    {% include copy-curl.html %}
+    
+    ```json
+    PUT /synonym_graph_index/_doc/1
+    { "content": "ssd is critical" }
+    ```
+    {% include copy-curl.html %}
+
+4. Search the non graph-aware index
+
+    ```json
+    POST /synonym_index/_search
+    {
+      "query": {
+        "match_phrase": {
+          "content": "solid state drive is critical"
+        }
+      }
+    }
+    ```
+    {% include copy-curl.html %}
+  
+    Expected result (no hits):
+    
+    ```json
+    {
+      "took": 13,
+      "timed_out": false,
+      "_shards": {
+        "total": 1,
+        "successful": 1,
+        "skipped": 0,
+        "failed": 0
+      },
+      "hits": {
+        "total": {
+          "value": 0,
+          "relation": "eq"
+        },
+        "max_score": null,
+        "hits": []
+      }
+    }
+    ```
+
+5. Search the graph-aware index
+
+    ```json
+    POST /synonym_graph_index/_search
+    {
+      "query": {
+        "match_phrase": {
+          "content": "solid state drive is critical"
+        }
+      }
+    }
+    ```
+    {% include copy-curl.html %}
+    
+    Expected result (1 hit): 
+    
+    ```json
+    {
+      "took": 9,
+      "timed_out": false,
+      "_shards": {
+        "total": 1,
+        "successful": 1,
+        "skipped": 0,
+        "failed": 0
+      },
+      "hits": {
+        "total": {
+          "value": 1,
+          "relation": "eq"
+        },
+        "max_score": 1.4384103,
+        "hits": [
+          {
+            "_index": "synonym_graph_index",
+            "_id": "1",
+            "_score": 1.4384103,
+            "_source": {
+              "content": "ssd is critical"
+            }
+          }
+        ]
+      }
+    }
+    ```
+
+The reason there is a hit using graph-aware token filter is because during the [`match_phrase`]({{site.url}}{{site.baseurl}}/query-dsl/full-text/match-phrase/) query additional subquery is generated using the token graph. Following diagram is demonstrating the token graph created using the graph-aware token filter:
```

Comment:
```suggestion
A hit occurs when using the graph-aware token filter because during the [`match_phrase`]({{site.url}}{{site.baseurl}}/query-dsl/full-text/match-phrase/) query, an additional subquery is generated using the token graph. The following diagram illustrates the token graph created by the graph-aware token filter.
```